diff --git a/lib/index.js b/lib/index.js
index 16ed6be8be8f555cc99096c2ff60954b42dc313d..a15e185202d63e0f67cde1bc80667e35ced4113d 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -99,7 +99,7 @@ function lodash(_ref) {
 
         var node = _ref3;
 
-        if ((0, _types.isModuleDeclaration)(node)) {
+        if ((0, _types.isImportDeclaration)(node) || (0, _types.isExportDeclaration)(node)) {
           isModule = true;
           break;
         }
